homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+
+const app = express();
+const port = 3000;
+
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  res.send("hello from backend to frontend!");
+});
+
+app.post("/weather", (req, res) => {
+  const cityName = req.body.cityName;
```

Comment:
We can use destructuring assignment here:
```suggestion
  const { cityName } = req.body;
```

Which will work the same way but later in future when you're receiving more than one key it's will be easier to destruct it that way Ex: `const {cityName, cityCode, etc..} = req.body;

Reply:
We should also add error handling here in case the `cityName` is empty. We don't want to show `undefined` message. 

Think of return a user friendly message to the user in case they forget to provide `cityName` some like: `CityName is required in the request body.`

To do that we have to check if the `cityName` is empty before sending the message like in line 15